openpype/modules/deadline/plugins/publish/submit_nuke_deadline.py

```diff
@@ -242,18 +244,18 @@ def payload_submit(self,
             "PYBLISHPLUGINPATH",
             "NUKE_PATH",
             "TOOL_ENV",
-            "OPENPYPE_DEV",
             "FOUNDRY_LICENSE"
         ]
+        # add allowed keys from preset if any
+        if self.env_allowed_keys:
+            keys += self.env_allowed_keys
+
         environment = dict({key: os.environ[key] for key in keys
                             if key in os.environ}, **api.Session)
         # self.log.debug("enviro: {}".format(pprint(environment)))
+
         for path in os.environ:
```

Comment:
I would just rename `path` to something differeny, like `key` as not all stuff beginning with `OPENPYPE_` is actually path.